### PR TITLE
#96 Change pattern to Text in FormattedText.

### PR DESCRIPTION
**Pull request by smallcreep**
As per #96

### CODE REVIEW

**Comment by 0crat** [comment]
@smallcreep I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/list/IterableAsList.java [hunk 1]
```diff
@@ -76,14 +77,18 @@ public IterableAsList(final Iterable<T> iterable) {
...
         if (index < 0 || index >= this.size()) {
-            throw new IndexOutOfBoundsException(
-                new FormattedText(
-                    "index=%d, bounds=[%d; %d]",
-                    index,
-                    0,
-                    this.size()
-                ).asString()
-            );
+            try {
```
Comment (yegor256) [inline src/main/java/org/cactoos/list/IterableAsList.java]: @smallcreep use `UncheckedText` instead, the code will be shorter and better

**Comment by yegor256** [comment]
@smallcreep one small comment

**Comment by smallcreep** [comment]
@yegor256 Thanks Yegor, I fixed it.

**Comment by yegor256** [comment]
@rultor try to merge

**Pull request merged**